ethclient/gethclient/gethclient_test.go

```diff
@@ -214,6 +215,49 @@ func testAccessList(t *testing.T, client *rpc.Client) {
 	if (*al)[0].StorageKeys[0] != common.HexToHash("0x0000000000000000000000000000000000000000000000000000000000000081") {
 		t.Fatalf("unexpected storage key: %v", (*al)[0].StorageKeys[0])
 	}
+
+	// error when gasPrice is less than baseFee
+	msg = ethereum.CallMsg{
+		From:     testAddr,
+		To:       &common.Address{},
+		Gas:      21000,
+		GasPrice: big.NewInt(1), // less than baseFee
+		Value:    big.NewInt(1),
+	}
+	al, gas, vmErr, err = ec.CreateAccessList(context.Background(), msg)
+	if errors.Is(err, core.ErrBlobFeeCapTooLow) {
```

Comment:
I don't get it, you are waiting for a specific error here, right?
Why are you checking for BlobFeeCap too low, which has nothing to do with the basefeecap?

Reply:
You are right. I just fix this code. (ErrFeeCapTooLow)

One thing is the err at this code, cannot be unwrapped because it was from RPC API response format. So I choose strings.Contains to get the error contains the ErrFeeCapTooLow.